pkg/stanza/docs/operators/container.md

```diff
@@ -31,16 +31,18 @@ will produce the following k8s metadata:
 
 ```json
 {
-  "attributes": {
-    "k8s": {
-      "container": {
-        "name": "kube-controller",
-        "restart_count": "1"
-      }, "pod": {
-        "uid": "49cc7c1fd3702c40b2686ea7486091d6",
-        "name": "kube-controller-kind-control-plane"
-      }, "namespace": {
-        "name": "some-ns"
+  "resource": {
+    "attributes": {
+      "k8s": {
+        "container": {
+          "name": "kube-controller",
+          "restart_count": "1"
+        }, "pod": {
+          "uid": "49cc7c1fd3702c40b2686ea7486091d6",
+          "name": "kube-controller-kind-control-plane"
+        }, "namespace": {
+          "name": "some-ns"
+        }
```

Comment:
Didn't we flatten these?

Reply:
I "wrongly" fixed a wrong left-over. Thank's!